openedx/features/partners/tests/test_helpers.py

```diff
@@ -32,13 +32,17 @@ def setUp(self):
         self.partner_user = PartnerUserFactory(user=self.user, partner=self.partner)
         self.course = CourseFactory.create()
 
-    def test_import_form_using_slug_with_valid_slug(self):
-        """
-        Test if form is available for a valid slug
-        :return : partner view
-        """
-        form = helpers.import_form_using_slug(self.PARTNER_SLUG)
-        self.assertIsNotNone(form)
+    '''
+    We need to comment this because we have already generalized the partners application
+    This test is not valid for now!
```

Comment:
👍

Reply:
@aliassad shouldn't we just remove it instead of commenting it out?

Reply:
I think we need it for future use

Reply:
Let's please not keep it commented just because we might be using it in future, if we have to use it in future we can always get it from git history? @aliassad WDYT?

Reply:
@ahmed-faraz46 fair point. But I guess commenting it would be better in the sense that whoever works on this file again, would see this commented test case and uncomment it, instead of we just totally forgetting that this ever existed. wdyt?

Reply:
Also @aliassad, let's update this to something like if we decide to keep it:
"At the time of commenting this, no directory with a form existed for a partner's custom requirements. Please uncomment this if such a form exists now which `import_form_using_slug` can make a use of."

Reply:
@mabdullahz  I totally agree, but this is not a lot of work which we may need to remember that was just a use of 'import_form_using_slug' which is still in our helper class and it has a nice comment for future help 👍